dockstore-webservice/src/main/resources/openapi.yaml

```diff
@@ -4346,6 +4346,32 @@ paths:
                   $ref: "#/components/schemas/DockstoreTool"
       security:
         - BEARER: []
+  "/users/{userId}/services":
+    get:
+      tags:
+        - users
+      summary: List all services owned by the logged-in user.
+      description: ""
+      operationId: userServices
+      parameters:
+        - name: userId
+          in: path
+          description: User ID
+          required: true
+          schema:
+            type: integer
+            format: int64
+      responses:
+        "200":
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/Workflow"
```

Comment:
Might run into problems from the UI side if there are properties that `Service` has that `Workflow` does not.  Same with BioWorkflow